test/src/test/java/jenkins/agents/WebSocketAgentsTest.java

```diff
@@ -99,9 +100,10 @@ public void smokes() throws Exception {
             ).stdout(System.out).start());
             r.waitOnline(s);
             assertEquals("response", s.getChannel().call(new DummyTask()));
+            assertNotNull(s.getChannel().call(new FatTask()));
             FreeStyleProject p = r.createFreeStyleProject();
             p.setAssignedNode(s);
-            p.getBuildersList().add(Functions.isWindows() ? new BatchFile("echo hello") : new Shell("echo hello"));
+            p.getBuildersList().add(Functions.isWindows() ? new BatchFile("echo hello") : new Shell("dd if=/dev/random count=1024 bs=200"));
```

Comment:
Probably safer to use `/dev/urandom` here if you're low on entropy?

Reply:
This line has been reverted by #4605 anyway

Reply:
⇒ https://github.com/jenkinsci/jenkins/pull/4710#discussion_r420079298